environment.yml

```diff
@@ -9,7 +9,7 @@ dependencies:
   - matplotlib
   - nbsphinx
   - numpy
-  - python > 3.6
+  - python >= 3.6
```

Comment:
Could you add the test in travis for python 3.6 please?

Reply:
done